src/main/scala/zio/redis/options/Shared.scala

```diff
@@ -1,6 +1,8 @@
 package zio.redis.options
 
 trait Shared {
+  sealed case class Limit(offset: Int, count: Int)
```

Comment:
shared by `SortedSets` and `Keys`

Reply:
It doesn't seem to be used in keys though.

Reply:
I made that change to implement the `sort` command but then realised that `sort` can't be introduced yet until the current design can accommodate it. I can move it back to `SortedSets` if `sort` is not viable.

Reply:
Already did it on interpreter.